src/components/organisms/Layout/index.types.ts

```diff
@@ -1,3 +1,3 @@
 export default interface LayoutProps {
-  children: JSX.Element | JSX.Element[]
+  children?: JSX.Element | JSX.Element[]
```

Comment:
Why the change?

Reply:
the `children` is no longer necessary, but before I completely delete the `index.types.ts` file I wanted to do some testing with BrowserRouter to understand how to make the layout work best. Anyway, now I’ve removed it